config/manager/kustomization.yaml

```diff
@@ -14,7 +14,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/netobserv/network-observability-operator
-  newTag: 1.0.1
+  newName: quay.io/kalmanmeth/network-observability-operator
+  newTag: test
```

Comment:
Could you please revert this before we merge @KalmanMeth ?

Reply:
done